src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -611,6 +620,22 @@ public static void GetFullPath_Windows_ValidExtendedPaths(string path)
             }
         }
 
+        [PlatformSpecific(TestPlatforms.Windows)]  // Tests Windows-specific paths
+        [Theory]
+        [InlineData(@"\\.\UNC\")]
+        [InlineData(@"\\.\UNC\server7")]
```

Comment:
I would definitely update the ones here that are "normal" length.  There might be some value in single character arguments such as `\\a\b`.  Fixing dead test runs because someone introduced a "server7" that takes forever to respond is something I've done several times. :)